Description: install translations in /usr/share/qtmobility/translations/
             instead of /usr/translations/
Author: Fathi Boudra <fabo@debian.org>
---
 translations/translations.pro |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/translations/translations.pro
+++ b/translations/translations.pro
@@ -99,7 +99,7 @@ isEmpty(vcproj) {
 }
 
 qmfiles.files = $$prependAll(LANGUAGES, $$OUT_PWD/qtmobility_,.qm)
-meego|maemo6 {
+meego|maemo6|linux-* {
     qmfiles.path = $${QT_MOBILITY_PREFIX}/share/qtmobility/translations
 } else {
     qmfiles.path = $${QT_MOBILITY_PREFIX}/translations
